### PR TITLE
chore: trigger CI on pushes to master alone

### PR DESCRIPTION
**Pull request by simoneb**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
----
 name: CI
-on: [push, pull_request]
+
+on:
```
Comment (micheleriva) [inline .github/workflows/ci.yml:3]: So did they fix it? A couple of years ago, this syntax was not valid (https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/5), and I was not aware that they fixed it

Comment (simoneb) [reply .github/workflows/ci.yml:3]: yeah this is what we use everywhere to avoid running CI multiple times on PRs

**Comment by simoneb** [comment]
@micheleriva if you're ok I'll go ahead and merge this

**Comment by micheleriva** [comment]
@simoneb yes absolutely

**Review by micheleriva — APPROVED** [review]

**Pull request merged**